lib/widgets/message_list.dart

```diff
@@ -70,6 +71,27 @@ class MessageListAppBarTitle extends StatelessWidget {
 
   final Narrow narrow;
 
+  Widget _buildStreamRow(ZulipStream? stream, String text) {
+    final icon = switch (stream) {
+      ZulipStream(isWebPublic: true) => ZulipIcons.globe,
+      ZulipStream(inviteOnly: true) => ZulipIcons.lock,
+      ZulipStream() => ZulipIcons.hash_sign,
+      null => null, // A null [Icon.icon] makes a blank space.
+    };
+    return Row(
+      mainAxisSize: MainAxisSize.min,
+      crossAxisAlignment: CrossAxisAlignment.baseline,
+      textBaseline: TextBaseline.alphabetic,
+      children: [
+        Padding(
+          // TODO(design): The vertical alignment of the stream privacy icon is ad hoc and eyeballed.
+          padding: const EdgeInsets.only(bottom: 4),
+          child: Icon(size: 16, icon)),
+        const SizedBox(width: 8),
+        Text(text),
```

Comment:
…Oh, hmm, but that would defeat the centering we talked about in https://github.com/zulip/zulip-flutter/pull/219#discussion_r1267322726. Hmm.

Reply:
Hmm, good catch! It looks like those tests were introduced in #223 — so they weren't there when I initially wrote this, and I hadn't rerun `flutter test` since rebasing the branch.

Reply:
(Also I guess I rarely encounter this in manual testing because I keep my font size set to small. Even the topic in your screenshot above fits without overflow, though just barely.)

Reply:
Looks like the solution is to use `Flexible` instead of `Expanded`.

(I got there from the docs on [that `mainAxisSize` property](https://api.flutter.dev/flutter/widgets/Flex/mainAxisSize.html) which I used to solve the previous issue; it had a warning effectively about how it can be defeated by an `Expanded` widget, but the warning also points to a way out.)